hail/src/main/scala/is/hail/io/fs/RouterFS.scala

```diff
@@ -19,8 +19,6 @@ class RouterFS(fss: IndexedSeq[FS]) extends FS {
 
   override def createCachedNoCompression(filename: String): PositionedDataOutputStream = lookupFS(filename).createCachedNoCompression(filename)
 
-  def openNoCompression(filename: String, _debug: Boolean = false): SeekableDataInputStream = lookupFS(filename).openNoCompression(filename, _debug)
```

Comment:
Was this just meant to delete the extra parameter?

Reply:
yep, fixed.